config/manager/controller_manager_config.yaml

```diff
@@ -17,3 +17,30 @@ spec:
   runCompletionFeed:
     port: 8082
     endpoints: []
+  defaultProviderValues:
```

Comment:
These are Provider config values that we didn't want to set on the Provider resources themselves (e.g. we don't want users to have to specify number of replicas and resource limits for their provider service deployments, that should be hidden).